llvm/test/Feature/Repo/repo_alias_aliasOfAlias.cpp

```diff
@@ -0,0 +1,21 @@
+// This test is Linux-only.
+
+
+// RUN: rm -rf %t.db
+// RUN: env REPOFILE=%t.db clang -c -O1 -target x86_64-pc-linux-gnu-repo %s -o %t.o
+// RUN: env REPOFILE=%t.db clang -c -O1 -target x86_64-pc-linux-gnu-repo %s -o %t1.o
+
+class base {
+public:
+  ~base();
+};
+
+class derived : base {
+  ~derived();
+};
+
+base::~base() = default;
+derived::~derived() = default;
+
+// CHECK-NOT: Alias must point to a definition
```

Comment:
Could this test check that declarations are created for both?

Reply:
Added the declarations check!